frontend/src/views/Queue.vue

```diff
@@ -11,8 +11,11 @@
             'https://www.google.com/maps/search/?api=1&query=<address>&query_place_id=' +
               storeId
           "
-          ><i class="fa fa-map-marker fa-2x" aria-hidden="true"></i
-        ></a>
+        >
+          <i class="fa fa-map-marker fa-2x" aria-hidden="true"></i>
+          <p>{{ travelTime }} </p>
+          <p class="error-sign" v-if="locationOn == false">Kindly enable your location for the best experience of our app</p>
```

Comment:
make a method to get this error message. Don't hardcode same error message on all pages.